### ISSUE TITLE
early/late in the day: what is the structure?

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#390]
- [EWT query](http://universal.grew.fr/?custom=6402272177d84)
- [GUM query](http://universal.grew.fr/?custom=6402274fc7f9f)

Sometimes early/late is treated as the head of the PP, other times as an advmod.

If the whole phrase functions as an adverbial, the advmod analysis may be preferable as it is often easier to omit the adverb:

- Early in the morning, I heard birds chirping.
- In the morning, I heard birds chirping.
- *Early, I heard birds chirping.

In a predicative context, it seems that either can be omitted:

- Our arrival was early in the morning.
- Our arrival was early.
- Our arrival was in the morning.

**Comment by nschneid** [comment]
CGEL p. 574: "Later in the day, the situation had improved slightly". Seems that "later" cannot be omitted here.

**Comment by amir-zeldes** [comment]
> In a predicative context, it seems that either can be omitted

I think for these I would prefer the PP to be the matrix modifier when both are supplied, because I feel like 'early in the morning' subsets 'in the morning', rather than the other way round (but if there is no way to tell, we may as well have an arbitrary guideline, so it's just as valid?)

> Later in the day

I think that's a different 'in', since it no longer serves the matrix spatio-temporal function (it's only relative to 'later')

**Comment by nschneid** [comment]
My gut reaction before was that "early" could be an advmod qualifying "in the morning" but I guess that's because "morning" is more informative than "early" in terms of locating the time of the event. "In the day" referring to a calendar day would be highly uninformative and therefore weird as a temporal modifier of an event, but it makes sense as a point of reference for the earlier vs. later dimension.

As a syntactic test, how about "right" modification—PPs generally allow it, AdvPs not so much (in my dialect):

- it happened right/*very in the morning
- it happened very/*right early

Combining them:
- it happened very early in the morning
- *it happened right early in the morning

suggests to me that "early in the morning" is an AdvP (or maybe a headless juxtaposition of AdvP and PP).

**Comment by nschneid** [comment]
In comparative constructions, it is important to recognize that "earlier" etc. licenses a than-PP or as-PP:

- Feel free to arrive earlier than the announced time.
   Feel free to arrive as early as you like.
- Feel free to arrive earlier in the morning than the announced time.
   Feel free to arrive as early in the morning as you like.
- *Feel free to arrive in the morning than the announced time.
   *Feel free to arrive as in the morning as you like.

While technically we could have a nonprojective dependency to attach the comparand to "earlier" (if it was taken to modify "in the morning", or be its sister), it seems cleaner to say that the adverb simply has two PP dependents.

**Comment by amir-zeldes** [comment]
Yeah, I think there's something to the analogy to "from X to Y", it seems to occupy an intermediate space where they're ostensibly two phrases, but can be 'packaged' as though they were just one. 

There is a phenomenon in German which reminds me of this: in German, famously, only one phrase can appear before a matrix clause verb, and it can be any one adjunct or argument, but almost never two. One rare exception is a kind of 'packaging' operation like this:

  * Ralf ist mit ihm auf der Bühne "Ralf is with him on the stage"
  * Mit ihm ist Ralf auf der Bühne "With him is Rald on the stage"
  * Auf der Bühne ist Ralf mit ihm "On the stage is Ralf with him"
  * Mit ihm auf der Bühne ist Ralf "With him on the stage is Ralf"

The last example should ostensibly not be possible, but such cases are attested. What seems to be crucial is that it is possible to cognitively pacakge "with him on stage" as some sort of single location or broader specification, answering the question "where is Ralf?" (as opposed to two questions, "where is Ralf?" and "who is he with?")
